modules/ding_dibs/includes/dibs.payment.inc

```diff
@@ -9,10 +9,8 @@
  */
 function ding_dibs_payment_get_url($amount, $params, $callback) {
   if (!empty($params['params']['debt_ids'])) {
-    $params['order_info_long'] = array(array(t('Payment of library dues:')));
-    foreach ($params['params']['debt_ids'] as $debt_id) {
-      $params['order_info_long'][] = array($debt_id);
-    }
+    $params['order_info_long'] = array();
+    $params['order_info_short'] = t('Library dues: ') . implode(', ', $params['params']['debt_ids']);
```

Comment:
Scrutinizer kaster en fejl: 
Translatable strings must not begin or end with white spaces, use placeholders with t() for variables 
Prøv:
t('Library dues: @value', array('@value' => implode(', ', $params['params']['debt_ids'])));

Reply:
Fixed :-)